config/default.ts

```diff
@@ -24,5 +24,18 @@ export default {
     port: process.env.PORT || 8082,
     txsHashesRequestLimit: 150,
   },
-  blockfrostProjectKey: process.env.BLOCKFROST || ""
+  blockfrostProjectKey: process.env.BLOCKFROST || "",
+  milkomeda: {
+    minimumValue: 10000000,
+    address: process.env.MILKOMEDA_ADDRESS || "addr_test1qq6gwl46frfwfk593pqjg39vym476s64n9cmykdl7lv7w32jdqlum6v4sc3l3w9nmuf3ean86n2y7pl83tpxl2qjdw8qmjljz2",
+    tokensPerNetwork: [{ 
+          name: "testnet",
+          tokens: [ {name: "ADA", policyId: ""}, {name: "TADA", policyId: ""} ] 
+        },
+        { 
+          name: "mainnet",
+          tokens: [ {name: "ADA", policyId: ""} ] 
```

Comment:
ADA should probably be an empty name and policyID (or probably it's even better to just exclude it from the list and just implicitly know ADA is supported)

This is because the "name" here means something very specific in the on-chain encoding of a user-created token. It's not a name that is meant to be displayed to the user (I know the name "name" is really bad)

Reply:
Shouldn't we consider a scenario where we don't have ADAs (for instance when we'll have SOL-EVM)? Although this may happen not as soon as Cardano-EVM deployment, so I agree with you, we can implicitly assume that ADA is supported.

Didn't know that "name" has already important meaning - thanks for explanation! I'll change it to something like `tokenName`, so it's more informative.